### DISCUSSION TITLE
FeathersJS example

### DISCUSSION THREAD

**Discussion by onizuka-dev** [vercel/vercel#5000] [Ideas] [answered]
Hello! I wonder if you guys can provide a `FeathersJS` example.

Thanks!

**Comment by mcsdevv** [comment, marked as answer]
Hello @ricardohs35, as FeathersJS is a framework intended for a running server environment we have no plans on adding an example.

**Reply by onizuka-dev** [reply]
Thanks!, I had to ask hehe
